src/main/java/net/irisshaders/lilybot/commands/support/UpdateDrivers.java

```diff
@@ -0,0 +1,54 @@
+package net.irisshaders.lilybot.commands.support;
+
+import com.jagrosh.jdautilities.command.SlashCommand;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+
+import java.awt.*;
+import java.time.Instant;
+
+public class UpdateDrivers extends SlashCommand {
+    public UpdateDrivers() {
+        this.name = "update-drivers"; // The name that will appear in the suggestion list. Must to be lower case
+
+        this.help = "Informs the user on how to update drivers."; // The description of your command
+
+        this.defaultEnabled = true; // Sets weather this command is enabled or disabled by default. If disabled you must specify a permission
+
+//        this.enabledRoles = new String[]{MainClass.ROLE};   Sets the roles that allow you to use the command. Ideally you will have a specified role
+//                                                              in you main class that is set with an id in a .env file. Not needed if this.defaultEnabled is true
+
+        this.guildOnly = false; // Sets weather the command can be used in dms too. Not Threads count as DMs because Discord is concern
+
+        this.botPermissions = new Permission[]{Permission.MESSAGE_WRITE}; // Set's the required permission for the command to work
+
+        this.botMissingPermMessage = "The bot can't write messages!"; // The error message shown to the user if permission(s) above aren't full filled
+    }
+
+    @Override
+    protected void execute(SlashCommandEvent event) {
+
+        User user = event.getUser(); // Get's the user that ran the command
+
+        MessageEmbed embed = new EmbedBuilder() // Creates a builder for the embed, name "builder" in this case
+                .setTitle("Update your Drivers.")
+                .setDescription(
+                        "This can be done in multiple ways, depending on your graphics card:\n\n" +
+                                "AMD: You can use AMD's Radeon Software to easily update your drivers, If you don't have it, download it here: https://www.amd.com/en/support\n\n" +
+                                "NVIDIA: You can use GeForce Experience to easily update drivers, If you dont have it, download it here: https://www.nvidia.com/Download/index.aspx?lang=us"
+                )
```

Comment:
Should probably add 
"Intel: You can use Intel DSA to easily update your drivers, if you don't have it, download it here: https://www.intel.com/content/www/us/en/support/detect.html"

Reply:
I agree, could also use text blocks instead of the repeated “\n”

Reply:
will add intel